internal/declarative/v2/reconciler.go

```diff
@@ -469,26 +458,19 @@ func (r *Reconciler) renderTargetResources(
 
 	targetResources, err := r.ManifestParser.Parse(spec)
 	if err != nil {
-		r.Event(obj, "Warning", "ManifestParsing", err.Error())
 		obj.SetStatus(status.WithState(shared.StateError).WithErr(err))
 		return nil, err
 	}
 
 	for _, transform := range r.PostRenderTransforms {
 		if err := transform(ctx, obj, targetResources.Items); err != nil {
-			r.Event(obj, "Warning", "PostRenderTransform", err.Error())
 			obj.SetStatus(status.WithState(shared.StateError).WithErr(err))
 			return nil, err
 		}
 	}
 
 	target, err := converter.UnstructuredToInfos(targetResources.Items)
 	if err != nil {
-		// Prevent ETCD load bursts during secret rotation
-		if !util.IsConnectionRelatedError(err) {
```

Comment:
Is this util function still used somewhere else now?

Reply:
yes, it's a critical function actually.